src/vsr/replica.zig

```diff
@@ -4569,6 +4569,17 @@ pub fn ReplicaType(
             self.nack_prepare_op = null;
         }
 
+        fn reset_quorum_prepare_ok(self: *Self) void {
+            // "prepare_ok"s from prior views are not valid, even if the pipeline entry is reused
+            // after a cycle of view changes.
```

Comment:
```suggestion
            // "prepare_ok"s from previous views are not valid, even if the pipeline entry is reused
            // after a cycle of view changes. In other words, when a view change cycles around, so
            // that the original primary becomes a primary of a new view, pipeline entries may be
            // reused. However, the pipeline's prepare_ok quorums must not be reused, since the
            // replicas that sent them may have swapped them out during a previous view change.
```